app/index.html

```diff
@@ -316,22 +316,83 @@ <h2>Board of Directors</h2>
                 <dt>Vice President</dt>
                 <dd>Andrew Stifel '91</dd>
                 <dt>Secretary</dt>
-                <dd>Christine McIntosh '14</dd>
-                <dt>Treasurer</dt>
                 <dd>Matthew Herndon '04</dd>
+                <dt>Treasurer</dt>
+                <dd>Meri Iannetti '99</dd>
                 <dt>General Board Members</dt>
-                <dd>Valerie Derose Leimer '05</dd>
                 <dd>Gabriel Ayache '07</dd>
-                <dd>Meri Iannetti '99</dd>
-                <dd>Matthew Laks '15</dd>
-                <dd>Nina Inamdar '94</dd>
                 <dd>Mark Belcher '80</dd>
+                <dd>Nina Inamdar '94</dd>
                 <dd>Joe Kirk '86</dd>
+                <dd>Matthew Laks '15</dd>
+                <dd>Valerie Derose Leimer '05</dd>
+                <dd>Christine McIntosh '14</dd>
                 <dt>Head Coach</dt>
                 <dd>Daria Schneider</dd>
                 <dt>Cornell University Director of Athletics</dt>
                 <dd>Andy Noel</dd>
                </dl>
+               <h2>Board Bios</h2>
+               <p><b>Barbara Todd Heiner</b> BA ’81, A&S, fenced under Coach Jean-Jacques Gillet and served as captain her senior year. She studied Psychology at Cornell and earned a master’s degree in Social Psychology from Rutgers University in 1987 and then worked in market research at HBO in New York before becoming a full-time mom and moving to Seattle in 1994.</p>

```

Comment:
Please use HTML entities instead of the raw characters. We can see that GH is highlighting this in red to tell us it's an error, for instance:

![image](https://user-images.githubusercontent.com/829827/37185975-fb2191b6-22f7-11e8-8275-17cd57654b8f.png)

Instead of `&`, we should write `&amp;`.

Here is a tool to convert automatically: https://mothereff.in/html-entities.

Reply:
I agree on the html entities, my bad on the &'s. Your tool also clued me in that we were using three different kinds of apostrophe (```&#x2018; &#x27; &#x2019;``` &#x2018; &#x27; &#x2019;) in the bios. I've updated them all to be &#x2019; to match the style I saw used on the Cornell Daily Sun. I made this change to the apostrophes that were in previous posts/headers as well.

For some reason I can't reply to your black bar image comment so I'll put that here too. I did black bars rather than css because only some images needed them. Now 2 of 3 of the images have them, so I can see the argument for moving it to css, but I don't want to delay pushing these bios and the the banquet registration link for that.

Reply:
Makes sense that we shouldn't delay to get that fix in. ✅